infra/scripts/fendermint.toml

```diff
@@ -40,9 +41,17 @@ docker run \
   --env FM_IPC__TOPDOWN__PARENT_HTTP_ENDPOINT=${PARENT_ENDPOINT} \
   --env FM_IPC__TOPDOWN__PARENT_REGISTRY=${PARENT_REGISTRY} \
   --env FM_IPC__TOPDOWN__PARENT_GATEWAY=${PARENT_GATEWAY} \
+  --env FM_IPC__TOPDOWN__EXPONENTIAL_BACK_OFF=5 \
+  --env FM_IPC__TOPDOWN__EXPONENTIAL_RETRY_LIMIT=5 \
+  --env FM_IPC__TOPDOWN__POLLING_INTERVAL=10 \
+  --env FM_ABCI__LISTEN__HOST=0.0.0.0 \
+  --env FM_ETH__LISTEN__HOST=0.0.0.0 \
```

Comment:
You shouldn't have to set these, they are part of the [Dockerfile](https://github.com/consensus-shipyard/fendermint/blob/37bcaa42d7a9b9941fce10b36f309ad9b124edb4/docker/ci.Dockerfile#L24-L25)

Reply:
I saw it, but unfortunately for some reason they weren't being picked up. I will test it again though, now that I we know what was happening with the connections. Maybe it was something else and I was lucky and I added both changes at the same time (which could definitely be the case with all the try and error :) )